source/slang/slang-syntax.h

```diff
@@ -304,22 +304,121 @@ namespace Slang
     // This fill assert-fail if the object doesn't represent a literal value.
     IntegerLiteralValue GetIntVal(RefPtr<IntVal> val);
 
-    // Represents how much checking has been applied to a declaration.
+        /// Represents how much checking has been applied to a declaration.
     enum class DeclCheckState : uint8_t
     {
-        // The declaration has been parsed, but not checked
+            /// The declaration has been parsed, but
+            /// is otherwise completely unchecked.
+            ///
         Unchecked,
 
-        // We are in the process of checking the declaration "header"
-        // (those parts of the declaration needed in order to
-        // reference it)
-        CheckingHeader,
+            /// Basic checks on the modifiers of the declaration have been applied.
+            ///
+            /// For example, when a declaration has attributes, the transformation
+            /// of an attribute from the parsed-but-unchecked form into a checked
+            /// form (in which it has the appropriate C++ subclass) happens here.
+            ///
+        ModifieredChecked,
```

Comment:
ModifieredChecked - is that a typo?

Reply:
Yeah, that should be `ModifiersChecked`.